vendor/engines/secure_rooms/spec/controllers/secure_rooms_api/scans_controller_spec.rb

```diff
@@ -11,7 +11,8 @@
 
     subject { response }
 
-    let(:card_reader) { create(:card_reader, tablet_token: "TABLETID") }
+    let(:secure_room) { create(:secure_room, :with_schedule_rule) }
+    let(:card_reader) { create(:card_reader, tablet_token: "TABLETID", secure_room: secure_room) }
```

Comment:
I decided not to add it to  the factory because it gets generated automatically. I chose to explicitly set it here so it's a little clearer the value we expect to show up in the json does.